A/AffineArithmetic/Compat.toml

```diff
@@ -0,0 +1,3 @@
+["0.1"]
+IntervalArithmetic = "0.15"
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.